src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -492,6 +498,8 @@ export class SavedObjectsClient {
           ...object.attributes,
         },
         {
+          ...options,
+          workspaces: [],
```

Comment:
Do we need this line?

Reply:
Yes, options contain a workspace field with some workspaces, otherwise the delete workspace field would fail.

Reply:
This line is not needed as we've already delete the workspaces field?

Reply:
Yes, I just want to make it clear.

Reply:
I can also delete this line.

Reply:
Deleted.